desktop/ui/src/main/java/org/datacleaner/connection/DatastoreDescriptors.java

```diff
@@ -87,6 +88,10 @@
             "Connect to a SugarCRM system", SugarCrmDatastore.class, SugarCrmDatastoreDialog.class,
             IconUtils.SUGAR_CRM_IMAGEPATH);
 
+    private static final DatastoreDescriptor DATAHUB_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl("Datahub",
+            "Connect to a HIquality Datahub", DatahubDatastore.class, DatahubDatastoreDialog.class,
```

Comment:
The name would be DataHub (capital H) both here and the line above.

I doubt we want to add the HIquality prefix to it since DC also does not have that.